src/sst/core/config.cc

```diff
@@ -322,404 +1025,68 @@ Config::parseCmdLine(int argc, char* argv[])
 
     /* Handle non-positional arguments */
     if ( optind < argc ) {
-        ok = setConfigFile(argv[optind++]);
+        ok = helper.setConfigFile(argv[optind++]);
         /* Support further additional arguments to be args to the model */
         while ( ok && optind < argc ) {
-            ok = setModelOptions(argv[optind++]);
+            ok = helper.setModelOptions(argv[optind++]);
         }
     }
 
     if ( !ok ) return -1;
 
     /* Sanity check, and other duties */
-    Output::setFileName(debugFile != "/dev/null" ? debugFile : "sst_output");
+    Output::setFileName(debugFile_ != "/dev/null" ? debugFile_ : "sst_output");
 
-    if ( configFile == "NONE" ) {
+    if ( configFile_ == "NONE" ) {
         cout << "ERROR: no sdl-file specified" << endl;
```

Comment:
Why does this use `cout` while everywhere else is `C` based?

Reply:
Because it was there long before most of the other code and hans't been touched since.  Changed it to fprintf.